install/host_preparation.adoc

```diff
@@ -444,12 +444,15 @@ management in RHEL Atomic Host.
 [[configuring-docker-overlayfs]]
 === Configuring OverlayFS
 
-OverlayFS is a type of union file system. It allows you to overlay one file system on top of another.
+OverlayFS is a type of union file system. OverlayFS enables you to overlay one file system on top of another.
 Changes are recorded in the upper file system, while the lower file system remains unmodified.
 
 xref:../scaling_performance/optimizing_storage.adoc#comparing-overlay-graph-drivers[Comparing the Overlay Versus Overlay2 Graph Drivers]
 has more information about the *overlay* and *overlay2* drivers.
 
+To use the *overlay2* driver, the lower layer must use the XFS file system. The
```

Comment:
Hey @mikemckiernan - Just FYI going forward that we should not be hard wrapping lines at 80 characters. Sentences and paragraphs should be allowed to wrap naturally.

See: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines